draft1.md

```diff
@@ -47,15 +52,18 @@ For each dimension `i` in the embedded space:
   Let V2_i be W_i projected in embedded space S_B
   Let V2_i be the i-th dimension of the transformation matrix T
 ```
-** *NOTE:** to convert $V1_i$ back into vocabulary space, we take the dot product of the embedded vector and the inverse of embedding matrix.
+** *NOTE:** to convert $V1_i$ back into vocabulary space, we take the product of the embedded vector and the inverse of the embedding matrix.
+(**EGP: Note note: dot product of a vector and a matrix does not exist, so I removed "dot"; or did you mean something special?**)
 
-*T* will end be of size *NxN*.
+*T* will be of size *NxN*.
 
-We will use the transformation *T* to measure how much the embedded space has changed between times *A* and *B*. To ensure that our measure is symmetric, we will calculate two transformation matrices, *T(A,B)* to go from *A* to *B* and *T(B,A)* to go from *B* to *A*. We call the element-wise product of these two matrices the divergence matrix:
+We will use the transformation matrix *T* to measure how much the embedded space has changed between times *A* and *B*. To ensure that our measure is symmetric, we will calculate two transformation matrices, *T(A,B)* to go from *A* to *B* and *T(B,A)* to go from *B* to *A*. We call the element-wise product of these two matrices the divergence matrix:
+(**EGP: Element-wise product? Why?**)
 
 $D(A,B) = T(A,B) * T(B,A)$
 
-It is important to mention that the divergence matrix will be equal the identity matrix if the space embeddings *S_A* and *S_B* are equivalent).
+It is important to mention that the divergence matrix will be equal to the identity matrix if the space embeddings *S_A* and *S_B* are equivalent).
+(**EGP: this is not true if the multiplication is element-wise! Only for a normal matrix-multiplication, because then $T(A,B) = T^{-1}(B,A)$**)
```

Comment:
Perhaps this needs clarification: 

`$T(X,Y)$` will be be the identity when `X` and `Y` are equivalent.  So for  `A` and `B` which are equivalent, `$T(A,B)$` will be the identity and `$T(B,A)` will be the identity, so `$D(A,B) = T(A,B) * T(B,A)$` will be the identity. 

The whole `element-wise` product is more of a trick, but I might need to re-think that part.